### ISSUE TITLE
Crash on reboot

### ISSUE THREAD

**Issue by rossjrw** [rossjrw/tars#237]
Bot boots fine, crashes on reboot, citing "No module named 'tars'".

Happens in #120 and #234.

**Comment by rossjrw** [comment]
I can reproduce this without rebooting by calling the main script directly instead of invoking tars as a module:

```
pipenv run python3 tars/__main__.py config/kipp.skipirc.conf
```

instead of the usual:

```
pipenv run python3 -m tars config/kipp.skipirc.conf
```

The former is the invocation as per `os.execl` which is used for the reboot command.

**Comment by rossjrw** [comment]
Closed in #238
